x-pack/plugins/apm/server/routes/traces/get_trace_items.ts

```diff
@@ -74,12 +75,21 @@ export async function getTraceItems(
     },
   });
 
-  const errorResponse = await errorResponsePromise;
-  const traceResponse = await traceResponsePromise;
+  const [errorResponse, traceResponse, outgoingSpanLinksSizeMap] =
+    await Promise.all([
+      errorResponsePromise,
+      traceResponsePromise,
+      getOutgoingSpanLinksSizeMap({ traceId, setup, start, end }),
```

Comment:
I think count is more appropriate here. Maybe `getOutgoingSpanLinksCountBySpanId`?